app/src/main/java/org/mozilla/fenix/search/awesomebar/AwesomeBarView.kt

```diff
@@ -210,6 +212,21 @@ class AwesomeBarView(
         return searchEngine
     }
 
+    /**
+     * Get a suggestions header if [currentEngineName] is the one of the default search engine.
+     *
+     * @param currentEngineName Name of the currently selected search engine.
+     */
+    private fun getSearchEngineSuggestionsHeader(currentEngineName: String?): String? {
```

Comment:
🔍 nit: we should just use `currentSearchEngine: SearchEngine?` as the parameter.  This way we're just comparing the `SearchEngine` not the name (even though we can assume the name is not the same between search engines)